.husky/pre-commit

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
-yarn lint-staged --verbose
\ No newline at end of file
+# yarn lint-staged --verbose
```

Comment:
can we keep this

Reply:
Strange, I remember after my first commit I restored everything. We can re-enable this for sure. I dont know why it was still commented out.